bundles/com.espressif.idf.ui/OSGI-INF/l10n/bundle.properties

```diff
@@ -38,4 +38,8 @@ category.name.1 = Espressif
 menu.label.1 = Espressif
 menu.mnemonic = E
 command.label.5 = Erase Flash
-view.name = Callers View
\ No newline at end of file
+view.name = Callers View
+command.name.10 = Install IDF Components
+command.label.6 = Install ESP-IDF Components
```

Comment:
Can we keep the same name `Install ESP-IDF Components`?

Reply:
Done

Reply:
Done